### ISSUE TITLE
Code runs in version 0.5.18 but not in 0.5.19

### ISSUE THREAD

**Issue by Mokhahle61354** [bukosabino/ta#144]
The below code can run on 0.5.18 but not on 0.5.19, I get the error.
```
import ta

df_pairs = df_pairs
df_pairs = ta.utils.dropna(df_pairs)
df_pairs = ta.add_all_ta_features(df_pairs, open="Open", high="High", low="Low", close="Close", volume="Volume", fillna=True)
```

**error:**
```
in jupyter notebook: ->single positional indexer is out-of-bounds.
in google colab: -> SyntaxError: unknown encoding for '/usr/local/lib/python3.6/dist-packages/ta/wrapper.py': future_fstrings.

IndexError                                Traceback (most recent call last)
<ipython-input-22-ea917a1c055b> in <module>
      3 df_pairs = df_pairs
      4 df_pairs = ta.utils.dropna(df_pairs)
----> 5 df_pairs = ta.add_all_ta_features(df_pairs, open="Open", high="High", low="Low", close="Close", volume="Volume", fillna=True)

```

**Comment by bukosabino** [comment]
It is really strange. I can run this function using v0.5.19.

Can you share with me your google colab document or share with me the output of this command in your google colab?

```
!pip freeze
```

**Comment by bukosabino** [comment]
Ok, I can reproduce the error.

I think we will come back to last version.

**Comment by bukosabino** [comment]
So, please, run this command on google colab:

```
!pip install -U ta
```

It should install the new version 0.5.20 fixing the bug.

Thank you for your advice!

Best, 
Dario

**Comment by monism123** [comment]
Hi, I mentioned this problem here https://github.com/bukosabino/ta/pull/142 
to fix it you have to restart the google colab run time, I think the future fstrings should be included only if the python version is <3.6
